plugin/evm/imports_test.go

```diff
@@ -38,7 +38,9 @@ func getDependencies(packageName string) (map[string]struct{}, error) {
 	}
 
 	// Start collecting dependencies
-	collectDeps(pkgs[0])
+	for _, pkg := range pkgs {
+		collectDeps(pkg)
+	}
```

Comment:
Why were we collecting the first pkg only??

Reply:
No idea. I don't think it was causing any problems as we directly referenced the pkgs itself. But I found this was not working with wildcards like `plugin/evm/...` etc to capture all subpackages.

Reply:
Oh because for a single package specified, this is always of length 1. I suppose using `...` can return multiple ones, why not the change then!